lib/Field/FieldRepeatableControls.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { bool, number } from 'prop-types';
+import { ButtonIcon } from '../ButtonNew/ButtonIcon/ButtonIcon';
+import { ButtonIconDanger } from '../ButtonNew/ButtonIcon/ButtonIconDanger';
+
+export function FieldRepeatableControls({ repeatPosition, isLastRepeat }) {
+  const label = `${repeatPosition < 9 ? '0' : ''}${repeatPosition + 1}`;
+
+  return (
+    <div className="flex justify-end">
+      {repeatPosition > 0 && (
+        <ButtonIconDanger name="trash" className="mx-1 repeat-icon" />
+      )}
+      {repeatPosition > 0 && (
+        <ButtonIcon name="arrowUp" className="mx-1 repeat-icon" />
+      )}
+      {!isLastRepeat && (
+        <ButtonIcon name="arrowDown" className="mx-1 repeat-icon" />
+      )}
+      <span className="w-1" />
```

Comment:
💚 Is this possible with padding rather than adding a new HTML element - empty tags are potentially confusing for screen reader users (conjecture here, David will know better!)

Reply:
@DavidHollins6 any input here?

The reason I went with a span is because I'm using flex and justify end, to align all the buttons and labels to the right. When I change text-dir to RTL the span is in the right place already. If I was to use a padding on the label, I'd have to swap from `pr-1` to `pl-1` depending on the text direction, and so I'd have to pass the `dir` down as a prop to the controls too. See what I mean here:

![Screenshot 2020-05-19 at 10 22 56](https://user-images.githubusercontent.com/3472717/82309545-2eb88480-99bb-11ea-9e80-79bdae55015a.png)

But I'm happy to remove the span and go with padding if that would be better?